frontend/src/components/teamsAndOrgs/orgUsageLevel.js

```diff
@@ -6,15 +6,16 @@ import messages from './messages';
 import { StatsCardContent } from '../statsCardContent';
 import { ProgressBar } from '../progressBar';
 import { usePredictYearlyTasks } from '../../hooks/UsePredictYearlyTasks';
-import { useOrganisationLevel, useGetLevel } from '../../hooks/UseOrganisationLevel';
+import { useOrganisationLevel, usePredictLevel } from '../../hooks/UseOrganisationLevel';
 
 export function OrganisationUsageLevel({ completedActions, orgName, type, userIsOrgManager }) {
   const [currentLevel, nextLevelThreshold] = useOrganisationLevel(completedActions);
   const percent = parseInt((completedActions / nextLevelThreshold) * 100);
   const yearPrediction = usePredictYearlyTasks(completedActions, new Date());
-  const levelPrediction = useGetLevel(yearPrediction);
+  const levelPrediction = usePredictLevel(yearPrediction, type);
 
   const showTierInfo = ['DISCOUNTED', 'FULL_FEE'].includes(type) && userIsOrgManager;
+  const showDiscountLabel = levelPrediction.tier !== 'free' && type === 'DISCOUNTED';
```

Comment:
Write a test to `OrganisationUsageLevel` to confirm that condition and to confirm we are showing the discounted value.